refactor/src/Cage.py

```diff
@@ -815,7 +818,7 @@ def update_deltas(
             new_males: np.int64,
```

Comment:
why this is np.int64 and new_infections is int?

Reply:
#84